recipes/gem3-mapper/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "3.6.1" %}
+{% set sha256 = "195f158691fe85bacc90b9cf848f203aeba1c56332516261da3774fee3f7ddbf" %}
+
+package:
+  name: gem3-mapper
+  version: {{ version }}
+
+source:
+  url: https://github.com/smarco/gem3-mapper/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - bzip2
+    - zlib
+  run:
+    - bzip2
+    - zlib
+
+test:
+  commands:
+          #    - gem-mappability -h 2>&1 |grep -e "-I <index_prefix>.*(mandatory)"
+    - gem-mapper 2>&1 | grep 'index|-I <index_file.gem>'
+    - gem-indexer -h 2>&1 | grep 'input|-i <input_file> (Multi-FASTA)'
+    - gem-retriever -h 2>&1 | grep 'index|-I <index_file.gem> (GEM archive)'
+
+about:
+  home: https://github.com/smarco/gem3-mapper
+  license: GPL3
```

Comment:
license_file: LICENSE could be added.
Can you maybe also include a DOI or bio.tools ID?

Reply:
Will do.
I'm adding the original paper. There is an entry in bio.tools/gemmapper, but it is currently pointing to an older version, not updated, version of the software. I will add it for now and ask the developer to try and update it.